scripts/checkdeps

```diff
@@ -127,6 +127,15 @@ case "${DISTRO}" in
           [libstdc++-static]=libstdc++-static
         )
       fi
+      if [[ ${FEDORA_VERSION} -ge 34 ]]; then
```

Comment:
The project has a preference for the single `[` version of test.

Please update the commit message subject to "scripts/checkdeps: fix dep error on fedora >= 34" or similar (the scripts/checkdeps is the missing part).

Thanks,